src/tests/suite/daos_epoch.c

```diff
@@ -222,7 +222,6 @@ test_snapshots(void **argp)
 	uuid_unparse(co_uuid, uuid_str);
 	MUST(cont_open(arg, uuid_str, DAOS_COO_RW | DAOS_COO_NOSLIP, &coh));
 	assert_int_equal(arg->co_info.ci_nsnapshots, 0);
-	assert_int_equal(cinfo.ci_lsnapshot, 0);
```

Comment:
Good catch, thank you. I think this should be replaced with assert_int_equal(arg->co_info.ci_lsnapshot, 0);

Also following the next call to cont_open(), on line 355 the code tests cinfo (but should test arg->co_info). So the ci_lsnapshot assertion after that call seems like it should be:
assert_int_equal(arg->co_info.ci_lsnapshot, snaps[snap_count-1]);

Reply:
Ok